Add IP addresses for Security Scanner tool (#6130)
magento/devdocs: src/guides/v2.2/cloud/live/live.md

## Set up Magento Security Scan Tool {#security-scan}

The Magento Security Scan Tool enables you to regularly monitor your store websites and receive updates for known security risks, malware, and out-of-date software. This is a free service available for all implementations and versions of {{site.data.var.ece}}. You access the tool through your [Magento Marketplace account](https://account.magento.com/customer/account/login).

*  Monitor your sites security status and applied security updates
*  Receive security updates and site-specific notifications

> For detailed instructions to set up and perform scans, see the [Magento User Guide](http://docs.magento.com/m2/ee/user_guide/magento/security-scan.html). Typically, you want to start using this tool as you enter user acceptance testing (UAT).

Reviewer: ```suggestion
See the [Magento User Guide](http://docs.magento.com/m2/ee/user_guide/magento/security-scan.html) for information about setting up and using the security scan tool. Typically, you start using this tool when you begin user acceptance testing (UAT).
```

> Each site you scan must be registered through the Magento Security Scan tab. The registration process requires you to accept Magento’s disclaimer prior to scanning. You control both scan scheduling and the authorization of personnel to be notified when each scan is complete. You can schedule scans for a specific, recurring date and time or on demand as required.

Reviewer: ```suggestion
Each site you scan must be registered through the Magento Security Scan tab. During the registration process, you must accept the Magento disclaimer before you can begin scanning.  You control both the schedule and authorizing the user to receive notifications when each scan is complete.  You can schedule scans for a specific, recurring date and time, or run a scan on demand as needed.
```

{%
include note.html
type='info'
content='The security scan tool uses the following public IP addresses:

```text
52.72.230.169
52.86.204.1
52.87.98.44
```

You must whitelist these IP addresses in your network firewall rules to allow the tool to scan your site.'

> %}

Reviewer: Wonder if it would be better to put the note higher up, or list it as a prerequisite step instead of a note, so that customers don't have to backtrack after going through registration process. Also, the person registering might not be the same person to whitelist the addresses, so putting at a prerequisite gives time to get the task completed.

Author: done

Reviewer 2: 1) agree with @meker12 . 
2) Add: Magento security scan tool posts requests to ports 80 and 443 only.
3) Also please add the following section: Magento security scanner uses several user agent strings to simulate real-life malware activity. You might see those user agents in your analytics or access logs:

Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:57.0) Gecko/20100101 Firefox/57.0
GuzzleHttp/6.3.3 curl/7.29.0 PHP/7.1.18
Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/37.0.2062.120 Safari/537.36
Visbot/2.0 (+http://www.visvo.com/en/webmasters.jsp;bot@visvo.com)

Author: done

## Scan your site

{:.procedure}
To scan your site:

1. Access your [Magento Marketplace account](https://account.magento.com/customer/account/login).
1. Click the Security Scan tab and select **Go to Security Scan**.

> 1. In the **Actions** column for the site, select **Run Scan**. A notification status displays the scheduled scan.

Reviewer 3: ```suggestion
1. In the _Actions_ column for the site, select **Run Scan**. A notification status displays the scheduled scan.
```

We've been using _italics_ for location and **bold** for step action. Definitely not a show-stopper.

{:.procedure}
To review the report:

1. When the report completes, a notification displays.
1. In the site row, select the report you want to view from the **Reports** column. The order is latest to oldest.

The report lists issues including Failed Scans, Unidentified Results, and Successful Scans. Each entry provides detailed information for the scan, a list of issues to investigate, and actions to take. Some of these actions may require downloading and installing security patches. You can add those to a development branch on your local workstation.

> Scan results include a general label that describes whether a site passed or failed, plus detailed information about the checks performed. "Failed" indicates that the website contains a serious vulnerability, while "Unidentified" suggests that a deeper review is required by your team or hosting provider to determine if further action is required. We also provide suggested remediation steps for each failed security test. Security scan results are protected and viewable only by the registered user. Only users designated in the site registration process receive scan completion notifications.

Reviewer: ```suggestion
Scan results include a label that describes scan pass or fail status with detailed information about the checks performed:
-  "Failed" indicates that the website contains a serious vulnerability
-  "Unidentified" suggests that a deeper review is required by your team or hosting provider to determine if further action is required.

The scan results also provide suggested remediation steps for each failed security test. Security scan results are protected and viewable only by the registered user. Only users designated in the site registration process receive scan completion notifications.
```

Reviewer: What are all possible statuses? Consider listing them all.

Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/37.0.2062.120 Safari/537.36
Visbot/2.0 (+http://www.visvo.com/en/webmasters.jsp;bot@visvo.com)
```

## Scan your site

{:.procedure}
To scan your site:

1. Access your [Magento Marketplace account](https://account.magento.com/customer/account/login).
1. Click the Security Scan tab and select **Go to Security Scan**.
1. In the _Actions_ column for the site, select **Run Scan**. A notification status displays the scheduled scan.

{:.procedure}
To review the report:

1. When the report completes, a notification displays.
1. In the site row, select the report you want to view from the **Reports** column. The order is latest to oldest.

The report lists issues including Failed Scans, Unidentified Results, and Successful Scans. Each entry provides detailed information for the scan, a list of issues to investigate, and actions to take. Some of these actions may require downloading and installing security patches. You can add those to a development branch on your local workstation.

Scan results include a label that describes scan pass or fail status with detailed information about the checks performed:

> 

Reviewer 4: Probably we have to add a description for "Pass" - it means the store "behaviors as expected" and it's responses on the certain requests are considered to be "correct"

> *  "Failed" indicates that the website contains a serious vulnerability.

Reviewer 4: I would avoid the word "serious" since, e.g. SSL TLS check has a minor "weight" and not SO serious... Also the failure is possible on the multiple reasons in addition to found vulnerability: undetected patch, malware found, server misconfiguration, vulnerable extension found.
Maybe instead of "...website contains a serious vulnerability..." we should put something like "... scanner fund a known vulnerability that should be reviewed and addressed accordingly..."
